src/encoded/static/components/matrix_chip_seq.js

```diff
@@ -0,0 +1,900 @@
+            <div className="matrix__presentation">
+                <Spinner isActive={spinnerActive} />
+                <div className={`matrix__label matrix__label--horz${!scrolledRight ? ' horz-scroll' : ''}`}>
+                    <span>biosample</span>
+                    {svgIcon('largeArrow')}
+                </div>
+                <div className="matrix__presentation-content">
+                    <div className="matrix__label matrix__label--vert"><div>{svgIcon('largeArrow')}{context.matrix.y.label}</div></div>
+                    {showOrganismRequest ? <SelectOrganismModal /> : null }
+                    <ChIPSeqTabPanel tabList={tabs} selectedTab={this.state.selectedTab} tabPanelCss="matrix__data-wrapper">
+                        <ChIPSeqTabPanel tabList={subTabsHeaders} selectedTab={selectedSubTab} tabPanelCss="matrix__data-wrapper" handleTabClick={this.subTabClicked} fontColors={fontColors}>
+                            {chIPSeqData && chIPSeqData.headerRow && chIPSeqData.headerRow.length !== 0 && chIPSeqData.dataRow && chIPSeqData.dataRow.length !== 0 ?
+                                  <div className="chip_seq_matrix__data" onScroll={this.handleOnScroll} ref={(element) => { this.scrollElement = element; }}>
+                                      <DataTable tableData={convertTargetDataToDataTable(chIPSeqData, selectedSubTab)} />
+                                  </div>
+                              :
+                                  <div className="chip_seq_matrix__warning">
+                                      { chIPSeqData && Object.keys(chIPSeqData).length === 0 ? 'Select an organism to view data.' : 'No data to display.' }
+                                  </div>
+                            }
+                        </ChIPSeqTabPanel>
+                    </ChIPSeqTabPanel>
+                </div>
+            </div>);
+    }
+}
+
+ChIPSeqMatrixPresentation.propTypes = {
+    context: PropTypes.object.isRequired,
+};
+
+ChIPSeqMatrixPresentation.contextTypes = {
+    navigate: PropTypes.func,
+    location_href: PropTypes.string,
+    session: PropTypes.object,
+    session_properties: PropTypes.object,
+};
+
+
+/**
+ * Container for ChIP-Seq Matrix page.
+ *
+ * @param {context}  context - Context object
+ * @returns
+ */
+const ChIPSeqMatrix = ({ context }) => {
+    const itemClass = globals.itemClass(context, 'view-item');
+
+    return (
+        <Panel addClasses={itemClass}>
+            <PanelBody>
+                <ChIPSeqMatrixHeader context={context} />
+                <ChIPSeqMatrixContent context={context} />
+            </PanelBody>
+        </Panel>
+    );
+};
+
+ChIPSeqMatrix.propTypes = {
+    context: PropTypes.object.isRequired,
+};
+
+ChIPSeqMatrix.contextTypes = {
+    location_href: PropTypes.string,
+    navigate: PropTypes.func,
+    biosampleTypeColors: PropTypes.object, // DataColor instance for experiment project
+};
+
+globals.contentViews.register(ChIPSeqMatrix, 'ChipSeqMatrix');
```

Comment:
I think this needs a blank line at the end.

Reply:
There is no blank link here but one in the file.

Reply:
In matrix_chip_seq.js, there was already a blank line in the end.